test/datasets_utils.py

```diff
@@ -114,26 +114,61 @@ def inner_wrapper(*args, **kwargs):
     return outer_wrapper
 
 
-def test_all_configs(test):
+def test_all_configs(*, remove_duplicates=True, strict=False):
```

Comment:
We don't seem to use `strcit=True` anywhere so I'd suggest to remove it to avoid dead code. we can introduce it when we need it ;)

Reply:
Same with `remove_duplicates`

Reply:
Yeah, your are right. I tend to overgeneralize stuff like this.